--- pyvenv/lib/python3.12/site-packages/ccv_generator/downloader.py	2026-07-26 20:59:34.824623374 -0400
+++ downloader.py	2026-07-26 21:03:33.195260308 -0400
@@ -5,6 +5,9 @@
 from platformdirs import user_cache_dir
 from datetime import datetime, timedelta
 
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
 APP_NAME = "CCCVGenerator"
 CACHE_DIR = Path(user_cache_dir(APP_NAME))
 CACHE_DIR.mkdir(parents=True, exist_ok=True)
@@ -55,7 +58,7 @@
         if etag:
             headers["If-None-Match"] = etag
 
-    response = requests.get(url, headers=headers, stream=True)
+    response = requests.get(url, headers=headers, stream=True, verify=False)
 
     if response.status_code == 200:
         # Save the new file
